package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "factory-girl",
   "description": "A factory library for Node.js and JavaScript inspired by factory_girl",
-  "main": "./src/index",
+  "main": "index.js",
```

Comment:
shouldn't this be 
```suggestion
  "main": "./src/index.js",
```

Reply:
nope... we are publishing the build folder... so the file structure is :
<img width="215" alt="Screenshot 2022-08-17 at 09 07 20" src="https://user-images.githubusercontent.com/52122087/185029348-e012b2a7-f8b2-49cb-a169-c230d90908f5.png">

Reply:
updated